src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -40,6 +40,7 @@
   <ItemGroup>
     <!-- Dependencies needed for Storage Providers -->
     <PackageReference Include="Azure.Core" Version="1.38.0" />
+    <PackageReference Include="Azure.Data.Tables" Version="12.8.3" />
```

Comment:
The latest version of this package is [12.9.0](https://www.nuget.org/packages/Azure.Data.Tables/12.9.0). Should we use that version?